examples/grid/PS_based/plot_PSC_grid_converter_6.9kVA.py

```diff
@@ -24,64 +20,44 @@
 
 # %%
 # Compute base values based on the nominal values (just for figures).
-
-base = mt.BaseValues(
-    U_nom=370, I_nom=15.5, f_nom=105.8, tau_nom=20.1, P_nom=6.9e3, p=2)
+base_values = mt.BaseValuesElectrical(
+    U_nom=400, I_nom=10, f_nom=50.0, P_nom=6.9e3)
 
 
 # %%
 # Configure the system model (grid model)
-rl_model = mt.InverterToInductiveGrid(L_g=73.8e-3, R_g=0)
-grid_model = mt.Grid(U_gN=np.sqrt(2/3)*400, w_g=2*np.pi*50)
+rl_model = mt.InverterToInductiveGrid(L_f = 6e-3, L_g=67.8e-3, R_g=0)
+grid_model = mt.Grid(w_N=2*np.pi*50)
 dc_model = None
 conv = mt.Inverter(u_dc=650)
-"""
-REMARK:
-    if you do not want to simulate any DC grid, you should define
-    dc_model = None. This would make the DC voltage constant, using the
-    value given in the converter model.
-    Do not forget also to activate/desactivate the dc-bus control
-"""
-    
+
+
 if dc_model == None:
     mdl = mt.GridCompleteModel(rl_model, grid_model, conv)
 else:
     mdl = mt.ACDCGridCompleteModel(
         rl_model, grid_model, dc_model, conv
         )
 
-
 pars = mt.PSCtrlPars(
-        L_f=73.8e-3,
+        L_f=6e-3,
         R_f=0,
-        C_dc = 1e-3,
         on_rf=False,
         on_v_dc=False,
-        S_base=base.P_nom,
+        S_base=6.9e3,
         i_max = 1.5,
         w_0_cc = 2*np.pi*5,
-        R_a = 4.64)
+        r_a = .2)
 ctrl = mt.PSCtrl(pars)
 
 # %%
 
-# Set the active and reactive power references
+# Set the active power reference that are inputs to the control system
```

Comment:
editorial feedback: power reference that are inputs -> power reference that is an input